vignettes/howtogenefilter.Rmd

```diff
@@ -0,0 +1,188 @@
+---
+title: "genefilter"
+author:
+- name: Robert, Gentleman
+  affiliation: Vignette Author's Institution
```

Comment:
Remove. The authors listed should be the same as those listed in the PDF plus the contributor.